### DISCUSSION TITLE
[Android|iOS] Soft keyboard pops up when resizing the image, blocking the UI of resize bars.

### DISCUSSION THREAD

**Discussion by SankethBK** [singerdmx/flutter-quill#825] [Mobile] [answered]
**This issue exists even in example app provided by flutter_quill. So please don't close this issue unless you really have to**

I am facing an issue related to soft keyboard, whenever i try to use the resize feature provided by default in flutter quill, soft keyboard pops up and blocks the UI of resizing bars. Is there any workaround for this. 

**The problem of soft keyboard popping up when trying to resize an image can also be seen in the video posted in this [issue](https://github.com/singerdmx/flutter-quill/issues/698)**

If there is no workaround for this, there should have been an option to completely remove this feature. Just like we can customize the options for selecting images through _link_  and _gallery_ options by passing callback to _mediaPickSettingSelector_ to _QuillToolbar.basic_

I'll try to describe the issue in images.

 1. When clicked on image, this window appears which has a resize option.
 [![resize option is selected here][2]][2]

 2. It has two resizing bars for height and width

[![resize bar appears][3]][3]

 3. Soft keyboard started appearing as soon as i click on one of the resizing bars
[![soft keyboard appearing][4]][4]

I want to disable this feature completely, if it isn't possible to fix it. please post any workaround for it, thank you!

This is my code. But this issue seems to be present in the example app given flutter_quill repo also.

```
class NoteCreatePage extends StatefulWidget {
  static String get route => '/note-create';

  const NoteCreatePage({Key? key}) : super(key: key);

  @override
  State<NoteCreatePage> createState() => _NoteCreatePageState();
}

class _NoteCreatePageState extends State<NoteCreatePage> {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      resizeToAvoidBottomInset: false,
      appBar: AppBar(
        title: Text("My Dairy"),
      ),
      body: Container(
        padding: EdgeInsets.only(
          top: 15.0,
          left: 10.0,
          right: 10.0,
          bottom: MediaQuery.of(context).viewInsets.bottom
        ),
        child: SingleChildScrollView(
          child: Column(
            children: [
              NoteTitleInputField(), 
              RichTextEditor2(key: GlobalKey()),
            ],
          ),
        ),
      ),  
    );
  }
}
 

class RichTextEditor2 extends StatefulWidget {
  const RichTextEditor2({Key? key}) : super(key: key);

  @override
  State<RichTextEditor2> createState() => _RichTextEditor2State();
}

class _RichTextEditor2State extends State<RichTextEditor2> {
  late  QuillController _controller;

    @override
    void initState() {
      _controller = QuillController.basic();
      super.initState();
    }


  @override
  Widget build(BuildContext context) {

    return Column(
      children: [
        QuillToolbar.basic(
          controller: _controller,
        ),
        Container(
          child: QuillEditor.basic(
            controller: _controller,

            readOnly: false, // true for view only mode
          ),
        )
      ],
    );
  }
}
```

  [1]: https://pub.dev/packages/flutter_quill
  [2]: https://i.stack.imgur.com/eSYy4.png
  [3]: https://i.stack.imgur.com/klerL.png
  [4]: https://i.stack.imgur.com/K3tUV.png

**Comment by putikorn** [comment]
I need workaround or bug fix too.

**Comment by singerdmx** [comment]
I am working on the fix. Please let me know if you have any info or suggestion.

**Comment by singerdmx** [comment]
@SankethBK I cannot repro this. I am using iOS simulator.

**Reply by SankethBK** [reply]
Please use android emulator or ios real device, as ios emulator doesn't forcefully pop soft keyboard even if textbox is focussed. It has to be manually opened by cmd + k

**Comment by singerdmx** [comment]
OK, I don't have those handy right now.
Could you try adding `editor!.hideToolbar();` in here?
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/122956/180713814-048d8725-bb81-4652-be96-ecd9d27cb322.png">

**Comment by singerdmx** [comment]
Oh, actually it is not the toolbar. Need to figure out what triggers keyboard open.

**Comment by singerdmx** [comment]
try this

<img width="514" alt="image" src="https://user-images.githubusercontent.com/122956/180716294-d77979b1-4209-4756-80f8-30587f7a925b.png">


<img width="587" alt="image" src="https://user-images.githubusercontent.com/122956/180716312-1611921f-d845-4420-a19c-7ccf4ac87dfb.png">

**Reply by SankethBK** [reply]
Please share the branch with your changes, I'll test it in sometime

**Reply by singerdmx** [reply]
Fixed in https://github.com/singerdmx/flutter-quill/commit/680a5db53110b8307dce47bced6145d66af4076d

**Comment by singerdmx** [comment, marked as answer]
Fixed in https://github.com/singerdmx/flutter-quill/commit/680a5db53110b8307dce47bced6145d66af4076d

**Comment by Mrgeh** [comment]
I have the same problem, how should I solve it? Sorry, I didn't understand your chat history, please help！

**Comment by SankethBK** [comment]
@Mrgeh It was fixed in the latest release, which version are you using?
